akka-http-core/src/test/scala/akka/http/impl/engine/ws/WebSocketIntegrationSpec.scala

```diff
@@ -69,12 +72,35 @@ class WebSocketIntegrationSpec extends AkkaSpec("akka.stream.materializer.debug.
       val binding = Await.result(bindingFuture, 3.seconds)
       val myPort = binding.localAddress.getPort
 
+      val completeOnlySwitch: Flow[ByteString, ByteString, Promise[Done]] = Flow.fromGraph(
+        new GraphStageWithMaterializedValue[FlowShape[ByteString, ByteString], Promise[Done]] {
+          override val shape: FlowShape[ByteString, ByteString] = FlowShape(Inlet("in"), Outlet("out"))
```

Comment:
this is just a small stage for this test, so I was lazy, but you are right, I will add names there